cypress/features/regression/experimental/search.feature

```diff
@@ -0,0 +1,71 @@
+Feature: Search component
+  I want to change Search component properties
+
+  Background: Open Search component page
+    Given I open "Experimental Search" component page
```

Comment:
```suggestion
    Given I open "Search" component page
```

Reply:
This has been addressed in commit 4ff060a5bf6bac1f57c0b3fab9e4daafe1322a0d

Reply:
this change brakes the build feature

Reply:
This change won't work

Reply:
fixed